src/lib.rs

```diff
@@ -51,7 +51,10 @@ pub async fn bootstrap(mut shutdown: broadcast::Receiver<()>, config: Configurat
 
     let analytics = analytics::initialize(&config, s3_client, geoip_resolver.clone()).await?;
 
-    let postgres = PgPoolOptions::new().connect(&config.postgres_url).await?;
+    let postgres = PgPoolOptions::new()
+        // https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2.setting-capacity.html#aurora-serverless-v2.max-connections
+        .max_connections(100)
```

Comment:
can't we have this as a config value defined from TF?

Reply:
Added in 453997a